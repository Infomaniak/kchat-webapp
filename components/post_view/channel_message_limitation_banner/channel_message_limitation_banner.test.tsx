// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
// See LICENSE.txt for license information.

import React from 'react';
import {shallow} from 'enzyme';

import {Provider} from 'react-redux';

import mockStore from 'tests/test_store';

import ChannelMessageLimitationBanner from './channel_message_limitation_banner';

describe('components/post_view/ChannelMessageLimitationBanner', () => {
    const baseProps = {
        olderMessagesDate: '2022-07-28 00:02:15.741000',
    };

    const store = mockStore({});

    test('should match snapshot', () => {
        const wrapper = shallow(
            <Provider store={store}>
                <ChannelMessageLimitationBanner{...baseProps}/>
            </Provider>,
        );
        expect(wrapper).toMatchSnapshot();
    });
});
