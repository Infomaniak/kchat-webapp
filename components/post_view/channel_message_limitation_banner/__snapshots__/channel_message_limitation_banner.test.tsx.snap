// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/post_view/ChannelMessageLimitationBanner should match snapshot 1`] = `
<ContextProvider
  value={
    Object {
      "store": Object {
        "clearActions": [Function],
        "dispatch": [Function],
        "getActions": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
      },
      "subscription": Subscription {
        "handleChangeWrapper": [Function],
        "listeners": Object {
          "notify": [Function],
        },
        "onStateChange": [Function],
        "parentSub": undefined,
        "store": Object {
          "clearActions": [Function],
          "dispatch": [Function],
          "getActions": [Function],
          "getState": [Function],
          "replaceReducer": [Function],
          "subscribe": [Function],
        },
        "unsubscribe": null,
      },
    }
  }
>
  <ChannelMessageLimitationBanner
    olderMessagesDate="2022-07-28 00:02:15.741000"
  />
</ContextProvider>
`;
